synapse/tests/test_cortex.py

```diff
@@ -4467,29 +4467,11 @@ async def test_cortex_nexslogen_off(self):
         '''
         Everything still works when no nexus log is kept
         '''
-        conf = {'nexslog:en': False, 'layers:logedits': True}
+        conf = {'nexslog:en': False}
         async with self.getTestCore(conf=conf) as core:
             self.len(2, await core.nodes('[test:str=foo test:str=bar]'))
             self.len(2, await core.nodes('test:str'))
 
-    async def test_cortex_logedits_off(self):
-        '''
-        Everything still works when no layer log is kept
-        '''
-        conf = {'nexslog:en': True, 'layers:logedits': False}
-        async with self.getTestCore(conf=conf) as core:
-            self.len(2, await core.nodes('[test:str=foo test:str=bar]'))
-            self.len(2, await core.nodes('test:str'))
-
-            layr = core.getLayer()
-            await self.agenlen(0, layr.syncNodeEdits(0, wait=False))
-            await self.agenlen(0, layr.syncNodeEdits2(0, wait=False))
-            # We can still generate synthetic edits though
-            ndedits = await alist(layr.iterLayerNodeEdits())
```

Comment:
Do we still have test for iterLayerNodeEdits kicking around?

Reply:
Yeah it is still used in some layer tests